src/java.base/share/classes/java/lang/StackWalker.java

```diff
@@ -101,18 +95,16 @@ public final class StackWalker {
      * {@link StackWalker}.
      *
      * <p> The information of a {@code StackFrame} available is determined by the
-     * {@linkplain Kind kind} of the stack walker.  Method information such as
+     * {@linkplain Option stack walking options} of a stack walker.
```

Comment:
Would this read better as "The information available from a {@code StackFrame} is determined ... "?

Reply:
What about "Stack walker options configure the stack frame information obtained by a StackWalker." - the first sentence from the Option javadoc and use it in the class spec of StackWalker and StackFrame.

Reply:
Sounds good!